todo/tests/integration/test_health_api.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.reverse import reverse
+from rest_framework.test import APISimpleTestCase, APIClient
+from rest_framework import status
+from unittest.mock import patch
+from todo.constants.health import AppHealthStatus, ComponentHealthStatus
+
+
+class HealthAPITests(APISimpleTestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    @patch(target="todo_project.utils.health.is_db_healthy", return_value=True)
+    def test_health_api_with_db_up_returns_success(self, mocked):
```

Comment:
are you sure, that's the way we write test name ..?

Reply:
Renamed all the tests methods